tests/acceptance/smoke.tsx

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable no-console */
+import { Meteor } from 'meteor/meteor';
+import { act, render } from '@testing-library/react';
+import React from 'react';
+import {
+  Location,
+  MemoryRouter,
+  NavigateFunction,
+  Route,
+  RouteObject,
+  Routes as ReactRouterRoutes,
+  useLocation,
+  useNavigate,
+} from 'react-router-dom';
+import fixtures from '../../imports/fixtures';
+import { GLOBAL_SCOPE } from '../../imports/lib/is-admin';
+import { stabilize, USER_EMAIL, USER_PASSWORD } from './lib';
+
+function enumeratePaths(routes: RouteObject[], prefix: string = '', acc: string[] = []): string[] {
+  routes.forEach((route) => {
+    if (route.children) {
+      enumeratePaths(route.children, `${prefix}${route.path}/`, acc);
+    }
+    acc.push(`${prefix}${route.path}`);
+  });
+  return acc;
+}
+
+if (Meteor.isClient) {
+  const Routes: typeof import('../../imports/client/components/Routes').default =
+    require('../../imports/client/components/Routes').default;
+  const { AuthenticatedRouteList, UnauthenticatedRouteList }: typeof import('../../imports/client/components/Routes') =
+    require('../../imports/client/components/Routes');
+
+  const location: React.MutableRefObject<Location | null> = { current: null };
+  const navigate: React.MutableRefObject<NavigateFunction | null> = { current: null };
+
+  const LocationCapture = () => {
+    location.current = useLocation();
+    navigate.current = useNavigate();
+    return null;
+  };
+
+  const TestApp = () => {
+    return (
+      <MemoryRouter>
+        <Routes />
+        <ReactRouterRoutes>
+          <Route path="*" element={<LocationCapture />} />
+        </ReactRouterRoutes>
+      </MemoryRouter>
+    );
+  };
+
+  const fixtureHunt = Object.keys(fixtures)[0];
+  const fixturePuzzle = fixtures[fixtureHunt].puzzles[0]._id;
+
+  describe('routes', function () {
+    before(async function () {
+      await Meteor.callPromise('test.resetDatabase');
+      await Meteor.callPromise('test.authentication.createUser');
```

Comment:
Given that we have a method for provisioning the first (admin) user from an empty database -- would it make sense to eliminate this method and replace it with a call to `provisionFirstUser`?  Then you don't have to mess with `test.authentication.addRole` either, and you're also getting some coverage benefit from exercising the actual first-user-setup flow.